.env

```diff
@@ -69,4 +69,5 @@ JAEGER_OLTP_PORT=4318
 TRACING_ENABLED=1
 
 # BREVO
-BREVO_ALLOWED_REDIRECT_URL=http://${DEV_DOMAIN:-localhost}${WORKAROUND_DOTENV_ISSUE}:${SITE_PORT}
\ No newline at end of file
+BREVO_ALLOWED_REDIRECT_URL=http://${DEV_DOMAIN:-localhost}${WORKAROUND_DOTENV_ISSUE}:${SITE_PORT}
+CAMPAIGNS_FRONTEND_URL=http://${DEV_DOMAIN:-localhost}${WORKAROUND_DOTENV_ISSUE}:${SITE_PORT} # doesn't work, TODO: add actual mailing frontend
```

Comment:
FYI
@RainbowBunchie added this here as well:
https://github.com/vivid-planet/comet-brevo-module/pull/41

Reply:
@raphaelblum 
I rebased onto this branch - so we will not get any problems with this env and also with the comet versions

Reply:
Thanks @RainbowBunchie